src/containerapp/azext_containerapp/custom.py

```diff
@@ -1244,16 +1247,27 @@ def containerapp_up(cmd,
                     user_assigned=None,
                     system_assigned=None,
                     custom_location_id=None,
-                    connected_cluster_id=None):
+                    connected_cluster_id=None,
+                    model_registry=None,
+                    model_name=None,
+                    model_version=None):
     from ._up_utils import (_validate_up_args, _validate_custom_location_connected_cluster_args, _reformat_image, _get_dockerfile_content, _get_ingress_and_target_port,
                             ResourceGroup, Extension, CustomLocation, ContainerAppEnvironment, ContainerApp, _get_registry_from_app,
                             _get_registry_details, _get_registry_details_without_get_creds, _create_github_action, _set_up_defaults, up_output,
-                            check_env_name_on_rg, get_token, _has_dockerfile)
+                            check_env_name_on_rg, get_token, _has_dockerfile, _validate_azml_args, _is_azml_app, _validate_azml_env, _set_azml_env_vars)
     from azure.cli.command_modules.containerapp._github_oauth import cache_github_token
     HELLOWORLD = "mcr.microsoft.com/k8se/quickstart"
     dockerfile = "Dockerfile"  # for now the dockerfile name must be "Dockerfile" (until GH actions API is updated)
+    ACA_AZML_MCR = "harrlistreamingcr.azurecr.io/azml:aca-general-img"
```

Comment:
This looks like an Private ACR?
Please use an public one

Reply:
this will be updated to an mcr image once we pushed it out.

Reply:
link to mcr pr: https://github.com/microsoft/mcr/pull/4115

Reply:
mcr image has been pushed and live now.